test/bdd/fixtures/demo/docker-compose-edge-components.yml

```diff
@@ -26,6 +27,22 @@ services:
     depends_on:
       - edv.example.com
 
+
+  rp.vcs.example.com:
+    container_name: rp.vcs.example.com
+    image: ${VCS_IMAGE}:${VCS_IMAGE_TAG}
+    environment:
+      - VC_REST_HOST_URL=0.0.0.0:8069
+      - VC_REST_HOST_URL_EXTERNAL=http://localhost:8069
+      - EDV_REST_HOST_URL=http://edv.example.com:8081
+      - BLOC_DOMAIN=${BLOC_DOMAIN}
```

Comment:
I guess this is the `TRUSTBLOC_DID_DOMAIN` / `DID_TRUSTBLOC_DOMAIN`...

Reply:
I will fix it next PR